myriad-scheduler/src/main/java/org/apache/myriad/MyriadModule.java

```diff
@@ -168,4 +172,16 @@ ExecutorCommandLineGenerator providesCLIGenerator(MyriadConfiguration cfg) {
     }
     return cliGenerator;
   }
+  
+  protected MyriadConfiguration generateMyriadConfiguration(String configFile) {
+    ObjectMapper mapper = new ObjectMapper(new YAMLFactory());
```

Comment:
HI Darin,

I looked at TaskUtils and there is no method that does this. The goal of this change together with the corresponding constructor parameter list change is to consolidate Guice-based object graph population within the MyriadModule class.